pkg/agent/run.go

```diff
@@ -99,17 +38,34 @@ var Prometheus bool
 // raw resource data of unstructuredList
 const schemaVersion string = "v2.0.0"
 
-const (
-	inClusterNamespacePath = "/var/run/secrets/kubernetes.io/serviceaccount/namespace"
-)
-
 // Run starts the agent process
 func Run(cmd *cobra.Command, args []string) {
+	logs.Log.Printf("Preflight agent version: %s (%s)", version.PreflightVersion, version.Commit)
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
-	config, preflightClient := getConfiguration()
 
-	if Profiling {
+	file, err := os.Open(Flags.ConfigFilePath)
+	if err != nil {
+		logs.Log.Fatalf("Failed to load config file for agent from: %s", Flags.ConfigFilePath)
+	}
+	defer file.Close()
+
+	b, err := ioutil.ReadAll(file)
+	if err != nil {
+		logs.Log.Fatalf("Failed to read config file: %s", err)
+	}
+
+	cfg, err := ParseConfig(b, Flags.StrictMode)
```

Comment:
**Post-merge comment:** Oops, it looks like I made a mistake here... It should be `Flags.VenafiCloudMode`, not `Flags.StrictMode`!